conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/BackwardsPackConverter.java

```diff
@@ -15,71 +15,91 @@
 import java.util.Objects;
 
 public class BackwardsPackConverter extends PackConverter {
-
     Path INPUT_DIR;
 
-    public BackwardsPackConverter(String from, String to, String light, boolean minify, Path input, boolean debug, PrintStream out) {
+    public BackwardsPackConverter(String from, String to, String light, boolean minify, Path input, boolean debug,
+            PrintStream out) {
         GsonBuilder gsonBuilder = new GsonBuilder().disableHtmlEscaping();
-        if (!minify) gsonBuilder.setPrettyPrinting();
+        if (!minify)
+            gsonBuilder.setPrettyPrinting();
```

Comment:
Style-wise, for single line if-statements, I prefer it on one line

Reply:
ah, formatter again + I find it easer to read that way